### DISCUSSION TITLE
DB Cleanup

### DISCUSSION THREAD

**Discussion by fun-git-user** [bpatrik/pigallery2#606] [Q&A]
Recently came to know about it and Thanks for this lightweight software, It is awesome. I would like to help in anyway if I can let me know how to get involved.

Now related to question, Recently I renamed some of photo files. But those photo with old name are somehow still showing up as item in folder view on page (item just empty space with missing thumbnail and photo) and noticed it still having entry in db (media_entity). I did ran indexing after renaming. Am I suppose to do something else as well?

**Comment by fun-git-user** [comment]
Just wrote a simple python code to cleanup DB media_entity table where file don't exist anymore. Looks like it happened only for one perticular folder while other folder are fine after photo filename rename.

**Comment by bpatrik** [comment]
The app should be able to see if the files in the folder got updated. It
uses the touch/modified date of the folder. If your file system does not
update that when you change a file inside the app won't detect the change.
You can try increasing the reindexing severity and see if that helps.

Patrik

-- Sorry for being brief, sent from my phone.
